io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,45 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+rename-icon: appimagepool
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - chmod +x /app/appimagepool/appimagepool
+      - cp -R appimagepool/usr/share /app/share
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - cp -R appimagepool/data /app/appimagepool/data
+      - sed -i 's/<id>appimagepool/<id>io.github.prateekmedia.appimagepool/' /app/share/metainfo/appimagepool.appdata.xml 
+      - mv /app/share/metainfo/appimagepool.appdata.xml /app/share/metainfo/io.github.prateekmedia.appimagepool.appdata.xml
```

Comment:
just use `rename-appdata-file`. 

Ideally it would have be named properly upstream.

Reply:
Do I also need to remove sed command, as it is renaming app id to flatpak id?

Reply:
I believe it edits the contents of the file also so you don't need `sed`. You can verify that.